dgp/lib/trajectory_ingestor.py

```diff
@@ -60,9 +60,17 @@ def import_trajectory(filepath, delim_whitespace=False, interval=0,
         Pandas DataFrame of ingested Trajectory data.
 
     """
-
     df = pd.read_csv(filepath, delim_whitespace=delim_whitespace, header=None,
-                     engine='c', na_filter=False, skiprows=skiprows)
+                     engine=engine, na_filter=False, skiprows=skiprows, sep=sep)
+    # if filepath[-12:-4] == 'TerraPOS':
+    #     print('You gave me a TerraPOS trajectory.')
+    #     df = pd.read_csv(filepath, delim_whitespace=delim_whitespace, header=None,
+    #                  engine='python', na_filter=False, skiprows=skiprows, sep='\s+')
+    #     # print(df.describe())
+    # else:
+    #     print('You gave me a Waypoint trajectory.')
+    #     df = pd.read_csv(filepath, delim_whitespace=delim_whitespace, header=None,
+    #                  engine='c', na_filter=False, skiprows=skiprows, sep=trajectory_delim)
```

Comment:
Should've mentioned this earlier, but remove commented code if it's not going to be used.